src/ensembl/package.json

```diff
@@ -59,6 +60,7 @@
     "redux": "4.0.1",
     "redux-devtools-extension": "2.13.8",
     "redux-thunk": "2.3.0",
+    "svg-react-loader": "0.4.6",
```

Comment:
Regarding @svgr/core and svg-react-loader: what functionality do they add compared to @svgr/webpack that we've already got?

(Also, if we were to add them, they would have gone to dev dependencies (`npm install --save-dev --save-exact blah`)

Reply:
I have actually removed them in the latest commit.